libs/core/langchain_core/tracers/event_stream.py

```diff
@@ -105,6 +106,22 @@ def __init__(
         self.send_stream = memory_stream.get_send_stream()
         self.receive_stream = memory_stream.get_receive_stream()
 
+    def _get_parent_ids(self, run_id: UUID) -> List[str]:
+        """Get the parent IDs of a run (non-recursively) cast to strings."""
+        parent_ids = []
+
+        run_info = self.run_map.get(run_id)
```

Comment:
lets us the dotted order string instead?

Reply:
created a parent map that should do essentially the same thing